sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/CxxPlugin.java

```diff
@@ -340,16 +341,6 @@
       .onQualifiers(Qualifiers.PROJECT, Qualifiers.MODULE)
       .index(3)
       .build(),
-      PropertyDefinition.builder(CxxCoverageSensor.FORCE_ZERO_COVERAGE_KEY)
```

Comment:
if you can find where that is found? at least from the tests ive done if you dont provide a coverage report it will not show coverage measures in server.